lib/cn/csched_sp3_work.c

```diff
@@ -453,7 +459,11 @@ sp3_work_wtype_length(
          * keys (i.e., k-compact).
          */
         if (runlen >= runlen_min) {
-            if (vwlen < VBLOCK_MAX_SIZE) {
+            if (wlen < VBLOCK_MAX_SIZE) {
+                *action = CN_ACTION_COMPACT_KV;
+                *rule = CN_RULE_LENGTH_WLEN;
+            }
+            else if (vwlen < VBLOCK_MAX_SIZE) {
```

Comment:
Put `else if` on same line as `}`

Reply:
I think this is harder to read. Doesn't make it seem like the `if` and `else if` are related at all.

Reply:
I find the clustter on the line above the else-if detracts from reading the predicate cuz it's buried in with the noise...